apostrophecms/apostrophe
# Breakpoint clear

## Summary

- Adds a Clear Breakpoint button when breakpoint prev is activated
- Moves Breakpoint controls to the left side of the context bar

## What are the specific steps to test this change?

https://github.com/user-attachments/assets/a79fcb17-bf7f-4adb-9aff-f163a9166816

1. Activate a breakpoint
2. See the X button appear
3. click it and be returned to full preview

**⚠️ NOTE**
The intended responsive effect of the breakpoint preview UI is that the shortcuts container will disappear at small screen sizes. But, due to an issue with the way the breakpoint preview webpack media query loader currently works, ALL media queries (including apos admin styles) get scoped to the breakpoint preview mode and therefore the shortcuts container disappears _whenever_ breakpoint preview is activated. This is _not_ the intended effect, but as other admin elements are already affected by this, this PR does not try to address it.

## What kind of change does this PR introduce?
*(Check at least one)*


## Make sure the PR fulfills these requirements:


If adding a new feature without an already open issue, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**
